drivers/display/display_rm68200.c

```diff
@@ -0,0 +1,455 @@
+	{0x4A, 0x00},
+	{0x4B, 0x00},
+	{0x4C, 0x00},
+	{0x4D, 0x00},
+	{0x4E, 0x00},
+	{0x4F, 0x00},
+	{0x50, 0x00},
+	{0x51, 0x00},
+	{0x52, 0x00},
+	{0x53, 0x00},
+	{0x54, 0x00},
+	{0x55, 0x00},
+	{0x56, 0x00},
+	{0x58, 0x00},
+	{0x59, 0x00},
+	{0x5A, 0x00},
+	{0x5B, 0x00},
+	{0x5C, 0x00},
+	{0x5D, 0x00},
+	{0x5E, 0x00},
+	{0x5F, 0x00},
+	{0x60, 0x00},
+	{0x61, 0x00},
+	{0x62, 0x00},
+	{0x63, 0x00},
+	{0x64, 0x00},
+	{0x65, 0x00},
+	{0x66, 0x00},
+	{0x67, 0x00},
+	{0x68, 0x00},
+	{0x69, 0x00},
+	{0x6A, 0x00},
+	{0x6B, 0x00},
+	{0x6C, 0x00},
+	{0x6D, 0x00},
+	{0x6E, 0x00},
+	{0x6F, 0x00},
+	{0x70, 0x00},
+	{0x71, 0x00},
+	{0x72, 0x20},
+	{0x73, 0x00},
+	{0x74, 0x08},
+	{0x75, 0x08},
+	{0x76, 0x08},
+	{0x77, 0x08},
+	{0x78, 0x08},
+	{0x79, 0x08},
+	{0x7A, 0x00},
+	{0x7B, 0x00},
+	{0x7C, 0x00},
+	{0x7D, 0x00},
+	{0x7E, 0xBF},
+	{0x7F, 0x02},
+	{0x80, 0x06},
+	{0x81, 0x14},
+	{0x82, 0x10},
+	{0x83, 0x16},
+	{0x84, 0x12},
+	{0x85, 0x08},
+	{0x86, 0x3F},
+	{0x87, 0x3F},
+	{0x88, 0x3F},
+	{0x89, 0x3F},
+	{0x8A, 0x3F},
+	{0x8B, 0x0C},
+	{0x8C, 0x0A},
+	{0x8D, 0x0E},
+	{0x8E, 0x3F},
+	{0x8F, 0x3F},
+	{0x90, 0x00},
+	{0x91, 0x04},
+	{0x92, 0x3F},
+	{0x93, 0x3F},
+	{0x94, 0x3F},
+	{0x95, 0x3F},
+	{0x96, 0x05},
+	{0x97, 0x01},
+	{0x98, 0x3F},
+	{0x99, 0x3F},
+	{0x9A, 0x0F},
+	{0x9B, 0x0B},
+	{0x9C, 0x0D},
+	{0x9D, 0x3F},
+	{0x9E, 0x3F},
+	{0x9F, 0x3F},
+	{0xA0, 0x3F},
+	{0xA2, 0x3F},
+	{0xA3, 0x09},
+	{0xA4, 0x13},
+	{0xA5, 0x17},
+	{0xA6, 0x11},
+	{0xA7, 0x15},
+	{0xA9, 0x07},
+	{0xAA, 0x03},
+	{0xAB, 0x3F},
+	{0xAC, 0x3F},
+	{0xAD, 0x05},
+	{0xAE, 0x01},
+	{0xAF, 0x17},
+	{0xB0, 0x13},
+	{0xB1, 0x15},
+	{0xB2, 0x11},
+	{0xB3, 0x0F},
+	{0xB4, 0x3F},
+	{0xB5, 0x3F},
+	{0xB6, 0x3F},
+	{0xB7, 0x3F},
+	{0xB8, 0x3F},
+	{0xB9, 0x0B},
+	{0xBA, 0x0D},
+	{0xBB, 0x09},
+	{0xBC, 0x3F},
+	{0xBD, 0x3F},
+	{0xBE, 0x07},
+	{0xBF, 0x03},
+	{0xC0, 0x3F},
+	{0xC1, 0x3F},
+	{0xC2, 0x3F},
+	{0xC3, 0x3F},
+	{0xC4, 0x02},
+	{0xC5, 0x06},
+	{0xC6, 0x3F},
+	{0xC7, 0x3F},
+	{0xC8, 0x08},
+	{0xC9, 0x0C},
+	{0xCA, 0x0A},
+	{0xCB, 0x3F},
+	{0xCC, 0x3F},
+	{0xCD, 0x3F},
+	{0xCE, 0x3F},
+	{0xCF, 0x3F},
+	{0xD0, 0x0E},
+	{0xD1, 0x10},
+	{0xD2, 0x14},
+	{0xD3, 0x12},
+	{0xD4, 0x16},
+	{0xD5, 0x00},
+	{0xD6, 0x04},
+	{0xD7, 0x3F},
+	{0xDC, 0x02},
+	{0xDE, 0x12},
+	{0xFE, 0x0E},
+	{0x01, 0x75},
+
+	/* Gamma Settings */
+	{0xFE, 0x04},
+	{0x60, 0x00},
+	{0x61, 0x0C},
+	{0x62, 0x12},
+	{0x63, 0x0E},
+	{0x64, 0x06},
+	{0x65, 0x12},
+	{0x66, 0x0E},
+	{0x67, 0x0B},
+	{0x68, 0x15},
+	{0x69, 0x0B},
+	{0x6A, 0x10},
+	{0x6B, 0x07},
+	{0x6C, 0x0F},
+	{0x6D, 0x12},
+	{0x6E, 0x0C},
+	{0x6F, 0x00},
+	{0x70, 0x00},
+	{0x71, 0x0C},
+	{0x72, 0x12},
+	{0x73, 0x0E},
+	{0x74, 0x06},
+	{0x75, 0x12},
+	{0x76, 0x0E},
+	{0x77, 0x0B},
+	{0x78, 0x15},
+	{0x79, 0x0B},
+	{0x7A, 0x10},
+	{0x7B, 0x07},
+	{0x7C, 0x0F},
+	{0x7D, 0x12},
+	{0x7E, 0x0C},
+	{0x7F, 0x00},
+
+	/* Page 0. */
+	{0xFE, 0x00},
+	{0x11, 0x00},
+};
+
+extern int mipi_dsi_dcs_write(const struct device *dev, const uint8_t *buf, uint8_t len);
```

Comment:
This looks like the approach "My board/display works, rest does not matter". I think there is no way around mipi-dsi driver API.

Reply:
Ok， I will fix this by open a repo for the rm6800 panel soon, with dts including the configuration, and get the info from this.